ui/src/app/components/help-sidebar.tsx

```diff
@@ -156,6 +158,19 @@ export const HelpSidebar = withUserProfile()(
       }
     });
 
+    componentDidMount(): void {
+      participantStore.subscribe(participant => this.setState({participant}));
+    }
+
+    componentDidUpdate(prevProps: Readonly<Props>): void {
+      const {helpContent, hideSidebar} = this.props;
+      // close the sidebar on each navigation excluding navigating between participants in cohort review
+      if (hideSidebar > prevProps.hideSidebar &&
+        !(helpContent === 'reviewParticipantDetail' && prevProps.helpContent === 'reviewParticipantDetail')) {
+        this.setState({activeIcon: undefined, sidebarOpen: false});
```

Comment:
If we have `hideSidebar` as a boolean, could we get rid of `sidebarOpen`? It seems redundant? I am a little confused by the interaction of `sidebarOpen` and `hideSidebar` here. It seems to me like `sidebarOpen` is the source of truth, but parent components also need to manage state? Which suggests maybe we should lift the state up and use `sidebarOpen` to trigger a callback?